cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules-external.yaml

```diff
@@ -36,3 +36,18 @@ spec:
       for: 5s
       labels:
         severity: critical
+  - name: cluster-services-alert.rules
+    rules:
+    - alert: ClusterObjectStoreState
+      annotations:
+        description: Cluster Object Store is in unhealthy state for more than 15s.
+          Please check Ceph cluster health or RGW connection.
+        message: Cluster Object Store is in unhealthy state. Please check Ceph cluster
+          health or RGW connection.
+        severity_level: error
+        storage_type: RGW
+      expr: |
+        ocs_rgw_health_status{job="ocs-metrics-exporter"} > 1
```

Comment:
Rook can't have a dependency on OCS, Rook upstream must be independent from OCS. How can we change this dependency in the other direction?

Reply:
I agree. Closing this. Will add OCS specific alerts in OCS.

Reply:
@anmolsachan which ocs-operator PR or issue is tracking this on OCS side?

Reply:
@mbukatov https://github.com/openshift/ocs-operator/pull/737. For now this is the one. We will send some more follow-ups on this.